.travis.yml

```diff
@@ -44,4 +45,7 @@ matrix:
         #       but since those are not supported by TravisCI, we need to install targets manually.
         - unzip "/c/Program Files (x86)/Windows Kits/10/Vsix/WDK.vsix" -d wdk
         - cp -R wdk/\$VCTargets/* "$MSVC/Common7/IDE/VC/VCTargets"
-        - MSBuild.exe haxm-windows.vcxproj //p:Configuration="Debug" //p:Platform="x64"
+        - export PROPS="//p:DefaultWindowsSDKVersion=10.0.17763.0 //p:SpectreMitigation=false"
+        - MSBuild.exe haxm.sln $PROPS //p:Configuration="Debug" //p:Platform="Win32"
+        - MSBuild.exe haxm.sln $PROPS //p:Configuration="Debug" //p:Platform="x64"
+        - build/tests/x64/haxm-tests.exe
```

Comment:
Ideally we should keep `Debug` and `Release` build output separate for `haxm-tests.vcxproj`, but that can be addressed later.

Reply:
Agreed. I'll prepare a PR for that.